metaflow/plugins/kfp/tests/flows/flow_triggering_flow.py

```diff
@@ -78,21 +63,10 @@ def test_trigger_and_wait(self):
             status: str = wait_for_kfp_run_completion(run_id=run_id, wait_timeout=180)
             print(f"Run Status of {run_id}:", status)
 
-            print("\nDemo that datastore of downstream job can be accessed")
```

Comment:
Is this test assertion captured elsewhere? Seems like there was no formal assertion but guessing querying the metadata store is an important test we should have somewhere 😅

Reply:
A similar test is kept, but simplified. `start_step.task.data.triggered_by` is a simplified way of retrieving downstream job data from datastore.

You are rigth that the previous "test" was more for demo purpose and isn't asserting anything, thus I changed it to assert on a passed in parameter `triggered_by`.